packages/block-library/src/legacy-widget/edit/control.js

```diff
@@ -361,7 +361,24 @@ async function encodeWidget( { idBase, instance, number, formData = null } ) {
 function isEmptyHTML( html ) {
 	const element = document.createElement( 'div' );
 	element.innerHTML = html;
-	return element.innerText.trim() === '';
+	return isEmptyNode( element );
+}
+
+function isEmptyNode( node ) {
+	switch ( node.nodeType ) {
+		case node.TEXT_NODE:
+			return node.nodeValue.trim() === '';
+		case node.ELEMENT_NODE:
+			if ( node.tagName === 'IMG' ) {
```

Comment:
Will there be the same issue with other types of element (video, svg)?

Reply:
Yes good point. Fixed in c07e34d.

Reply:
Cool. 👍